app/views/unattended/provisioning_templates/snippet/kickstart_network_interface.erb

```diff
@@ -0,0 +1,100 @@
+<%#
+name: kickstart_network_interface
+model: ProvisioningTemplate
+snippet: true
+model: ProvisioningTemplate
+kind: snippet
+description: |
+  Generates network directive for a given interface. It is not expected to be used directly.
+-%>
+<%#
+  # Variables: iface, host, use_slaac, static, static6
+-%>
+<%if @iface.managed? -%>
+<%
+network_options = []
+nameservers = []
+subnet4 = @iface.subnet
+subnet6 = @iface.subnet6
+
+# device and hostname
+if @iface.bond?
+  network_options.push("--device=#{@iface.identifier}")
+else
+  network_options.push("--device=#{@iface.mac || @iface.identifier}")
+end
+network_options.push("--hostname #{@host.name}")
+
+# single stack
+if subnet4 && !subnet6
+  network_options.push("--noipv6")
+elsif !subnet4 && subnet6
+  network_options.push("--noipv4")
+end
+
+# dual stack MTU check
+raise("IPv4 and IPv6 subnets have different MTU") if subnet4 && subnet6 && subnet4.mtu.present? && subnet6.mtu.present? && subnet4.mtu != subnet6.mtu
```

Comment:
I think we should implement this check in `@iface.mtu` instead. Perhaps not in this PR because I don't know how big the implications are.

Reply:
Created an issue: https://projects.theforeman.org/issues/37216